TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,154 @@
+   marks (e.g. Knative Serving, Knative Core, etc).
+
+1. Only the Trademark Committee can redefine its own charter, membership, and
+   authority over Knative Trademark, conformance, and brand decisions.
+
+It is expected that the Trademark Committee will delegate one or more marks to
+the steering committee to provide branding for nascent efforts which are not yet
+subject to conformance and third-party licensing.
+
+## Delegated authority
+
+The Trademark Committee may choose to create and delegate its authority to other
+committees as-needed. This may be used (for example) to create sub-committees
+for specific technical or marketing questions related to conformance or joint
+marketing.
+
+The Trademark Committee may delegate enforcement authority for one or more marks
+to other Knative committees (such as the steering committee) to provide branding
+for nascent efforts which are not yet subject to conformance and third-party
+licensing. This delegation may be further delegated transitively, as needed.
+
+## Committee Meetings
+
+The committee meets as-needed. Meetings are held online.
+
+Given the private nature of some of these discussions (e.g. trade secrets, legal
+and compliance concerns, etc.) some meetings are held in private.
+
+Meeting notes are available to members of the
+[knative-dev mailing list](https://groups.google.com/forum/#!forum/knative-dev)
+(note: these are accessible to members of the knative-users list as well).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as
+[issues in the community repo](https://github.com/knative/community/issues), and
+the trademark committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee Members
+
+| Member                   | Organization | Profile |
+| ------------------------ | ------------ | ------- |
+| Unknown - to be selected | Google       | Unknown |
+| Unknown - to be selected | IBM/RedHat   | Unknown |
+| Unknown - to be selected | VMware       | Unknown |
+
+Committee members for bootstrap term will be chosen during the Steering
+Committee election process.
+
+## Decision process
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding assignment of the mark, charter,
+conformance, and scope.
+
+Decisions requiring a vote include: issuing written policy, amending existing
+written policy, official responses to publicly raised issues, or any other
+decisions that are in the scope of the Trademark Committee’s charter.
+
+Voted decisions pass when all members vote to approve the decision, or a
+majority approves and the rest of the members abstain.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it. Vetoes by
```

Comment:
Just for my understanding: For a committee of 3, there must be at least 2 members present (quorum) and 2 members must support the decission. If only 2 members are voting, then the third member can veto within 5 days (if needed). If 3 members have voted, then my question is: Do all 3 have to support the decission ? If 2 support it and 1 not, isn't this then automatically already a veto ? (or is it possible to abstain a vote, even when present). So I understand the "may pass" wording as that 2 supporting votes are required but not necessarily sufficient. Is this correct ?

Reply:
I think a veto is separate from a vote -- for a committee of 3, at least 2 must be present and at least 2 at the meeting must be positively inclined (vote in agreement). The third would be allowed to abstain; I'm not sure whether the committee (which is supposed to be self-governing) would need to distinguish between a "object that can be overridden" and a "veto" with separate types of votes.